internal/environments/environments_test.go

```diff
@@ -181,45 +187,44 @@ func (suite *EnvironmentTestSuite) TestIntegrationEnvironmentGetByName() {
 
 func (suite *EnvironmentTestSuite) TestIntegrationEnvironmentListAll() {
 	suite.Run("ListAll returns nothing", func() {
-		testutils.Cleanup(suite.T(), suite.testApp.db)
 
 		foundEnvironments, err := suite.testApp.Environments.ListAll()
 
-		assert.Equal(suite.T(), len(foundEnvironments), 0)
+		assert.GreaterOrEqual(suite.T(), len(foundEnvironments), 0)
```

Comment:
wouldn't this pass incorrectly if ListAll returned 1 or more?

Reply:
oh sorry, I was meaning some wierd unexpected corner case where we end up with some strange number of new entries (not specifically checking for dupes, agree w/ your assessment there on mixing concerns), I just couldn't come up with a good example becuase.... wierd unexpected corner cases

Reply:
> That said there are some assumptions we had in the list all tests previously that are no longer valid. The main one being that the db is empty. It could be completely empty but it might not.

For my own edification: why is this the case if all db changes for each test happen in a transaction that is rolled back?